
# Find an *available* specific part of a given type

string type

---

string part_name

bool success
string message


